Doc/library/pathlib.rst

```diff
@@ -1003,6 +1003,10 @@ call fails (for example because the path doesn't exist).
       >>> target.open().read()
       'some text'
 
+   The target path may be either absolute or relative. If given a relative
+   path, it will be relative to the current working directory, *not* the
+   directory that the file is in.
+
```

Comment:
Perhaps a bit more concise?

"The target path may be absolute or relative. Relative paths are interpreted relative to the current working directory, *not* the directory of the Path object."

Reply:
Done.